### DISCUSSION TITLE
Welcome to billiard Discussions!

### DISCUSSION THREAD

**Discussion by auvipy** [celery/billiard#352] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fedeesku** [comment]
Hi @auvipy,

I came across the Billiards library as part of reading some of the celery documentation. I want to learn more about its capabilities but the read the docs link on Billiards sends me to what looks to be the standard multiprocessing library -  https://billiard.readthedocs.io/en/latest/. Can you please point me to the Billiards documentation or correct me if I am mistaken?

Thank you,

Federico
